clients/cobol-lsp-vscode-extension/src/extension.ts

```diff
@@ -52,7 +52,12 @@ const API_VERSION: string = "1.0.0";
 
 function initialize(context: vscode.ExtensionContext) {
   // We need lazy initialization to be able to mock this for unit testing
-  vscode.workspace.fs.createDirectory(context.globalStorageUri);
+  try {
+    vscode.workspace.fs.createDirectory(context.globalStorageUri);
```

Comment:
This should be awaited.

Reply:
resolved at : 45f48cc107eeaa09d23e59ae34b7281e41d5acf2

Reply:
resolved at : f69550d9b787df8b21c589d01c11a665a06ea304